artifactory/commands/buildinfo/publish_test.go

```diff
@@ -67,3 +72,34 @@ func TestExcludePasswordsPattern(t *testing.T) {
 		t.Error("expected:", expected, "got:", filteredKeys)
 	}
 }
+
+func TestPrintBuildInfoLink(t *testing.T) {
+	var linkTypes = []struct {
+		artVersion    int
+		buildTime     time.Time
+		buildInfoConf artifactoryUtils.BuildConfiguration
+		serverDetails config.ServerDetails
+		expected      string
+	}{
+		{5, time.Now(), artifactoryUtils.BuildConfiguration{BuildName: "test", BuildNumber: "1", Module: "6", Project: "cli"},
```

Comment:
```time.Now()``` shouldn't be called multiple times during this test, because it may change and fail the test.